src/app/data/routes.js

```diff
@@ -84,7 +84,7 @@ export const routes = [
 				name: '/store/products',
 				title: __( 'Products & Services', 'wp-plugin-bluehost' ),
 			},
-			NewfoldRuntime.hasCapability( 'hasYithExtended' )
+			NewfoldRuntime.hasCapability( 'hasYithExtended' ) || (NewfoldRuntime.hasCapability('canAccessGlobalCTB') && (NewfoldRuntime.sdk.ecommerce.brand_settings.brand === "bluehost"))
```

Comment:
Let's remove the brand check here. Given that we're adding this code to the Bluehost plugin, it shouldn't be necessary, right?

Reply:
@wpscholar , I removed the condition to check for Bluehost brand. Can you please confirm if code base for Bluehost India and Bluehost US as same as this should only visible to Bluehost US not Bluehost India.

Reply:
@ramyakrishnai Ah, if there is a need to only show this for Bluehost US and not Bluehost India or Asia, then we probably need a check for that. However, I recently discovered the `Brands.php` file, and I don't see how it could possibly be detecting Bluehost India correctly, so I am quite certain it all defaults to Bluehost US anyway. We have that on the list as a follow-up and perhaps a better way of identifying brands in general.

Reply:
Resolving this here, since we have this Brands.php on our radar now.